Directory.Build.targets

```diff
@@ -10,16 +10,10 @@
     <FsYaccPath>$(ArtifactsDir)/bin/fsyacc/$(Configuration)/$(FSharpNetCoreProductDefaultTargetFramework)/$(NETCoreSdkPortableRuntimeIdentifier)/fsyacc.dll</FsYaccPath>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(UnitTestType)' == 'nunit'">
-    <PackageReference Include="NUnit" Version="$(NUnitVersion)" />
-    <PackageReference Include="NUnit3TestAdapter" Version="$(NUnit3TestAdapterVersion)" />
-    <PackageReference Include="NunitXml.TestLogger" Version="$(NunitXmlTestLoggerVersion)" />
-  </ItemGroup>
-
   <ItemGroup Condition="'$(UnitTestType)' == 'xunit'">
```

Comment:
This has a different value only [here](https://github.com/dotnet/fsharp/blob/main/tests/service/data/CSharp_Analysis/CSharp_Analysis.csproj#L7).

Quite sure it can be removed there so things could be further simplified - but that's not the point of the PR.